<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mnwMain</class>
 <widget class="QMainWindow" name="mnwMain">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>455</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Chrubix</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lstTasks">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>256</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExecute">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Changes that require a reboot</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Greeter settings</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pteLxdmSettings">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>311</width>
      <height>211</height>
     </rect>
    </property>
    <property name="plainText">
     <string>
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveLxdmSettings">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save LXDM Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menuChange_Password">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Change Passwords</string>
     </property>
     <addaction name="actionChangePwDisk"/>
     <addaction name="actionChangePwRoot"/>
     <addaction name="actionChangePwBoom"/>
    </widget>
    <addaction name="actionBoot_Settings"/>
    <addaction name="menuChange_Password"/>
    <addaction name="actionEncryption"/>
    <addaction name="actionWhitelist"/>
    <addaction name="actionObfuscation"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionChangePwDisk">
   <property name="text">
    <string>Disk</string>
   </property>
  </action>
  <action name="actionChangePwRoot">
   <property name="text">
    <string>Root</string>
   </property>
  </action>
  <action name="actionChangePwBoom">
   <property name="text">
    <string>Boom</string>
   </property>
  </action>
  <action name="actionDecrypt">
   <property name="text">
    <string>Decrypt Disk</string>
   </property>
  </action>
  <action name="actionWhitelist_ON">
   <property name="text">
    <string>Create Whitelist</string>
   </property>
  </action>
  <action name="actionEncryption">
   <property name="text">
    <string>Decrypt Root Partition</string>
   </property>
  </action>
  <action name="actionWhitelist">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hardware Whitelist</string>
   </property>
  </action>
  <action name="actionObfuscation">
   <property name="text">
    <string>Obfuscate Filesystem</string>
   </property>
  </action>
  <action name="actionBoot_Settings">
   <property name="text">
    <string>Boot Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
